strangerstudios/paid-memberships-pro
# Health: catch addons installed in wrong folder

### All Submissions:






### Changes proposed in this Pull Request:

Sometimes you download an addon from github and you don't remove the _-branch_ suffix.

This is from a conversation with Kim White; sometimes you are investigating a weird situation on a customer's site and you don't see that there's an addon installed with that wrong folder name, which prevents to be shown into PMPro > Add Ons > Installed list. Also prevents it to be updated.

Thanks to this PR, you will immediately see it into Site Health. Can be a starting point to show a warning into PMPro > Add Ons.
![Screenshot 2022-05-09 at 16 49 09](https://user-images.githubusercontent.com/636911/167436299-87a17f82-b417-49df-9a77-1a6fd6a10249.png)


### How to test the changes in this Pull Request:

1. Download a copy of an addon from github and install
2. Check Site Health > Info > PMPro section

### Other information:
